packages/app/src/styles/theme/_apply-colors.scss

```diff
@@ -1,9 +1,10 @@
 @use '../variables' as var;
-@use '../bootstrap/init' as *;
 @use '../mixins';
 @use './mixins/tables'; // comment out and use _reboot-bootstrap-tables instead -- 2020.05.28 Yuki Takei
 @use '../atoms/mixins/code';
 
+@import '../bootstrap/init';
```

Comment:
```
@use '../bootstrap/init' as * with ($input-focus-color);
```
上記のように`with`を使って 変数を上書きしようとすると、
一度読み込んだモジュールが with 使用で再度読み込まれている、という旨のエラーが出てしまうので、
```
ruleSet[1].rules[2].oneOf[9].use[2]!../../node_modules/next/dist/build/webpack/loaders/resolve-url-loader/index.js??ruleSet[1].rules[2].oneOf[9].use[3]!../../node_modules/next/dist/compiled/sass-loader/cjs.js??ruleSet[1].rules[2].oneOf[9].use[4]!./src/styles/style-themes.scss
SassError: This module was already loaded, so it can't be configured using "with".
  ╷
2 │ @use '../bootstrap/init' as * with ($input-focus-color: red);

```

@import でスタイルシートを読み込むことにしました。

Reply:
> @import でスタイルシートを読み込むことにしました。

で済ませたくないのはある

- `as *` だからエラーになるんだと思う
    - `as *` な行と `as bs with ...` な行を両方用意して、`bs` ネームスペースで mixin を使えばうまいこといかないかな…
- あるいは新たな reboot 系 scss の作成

Reply:
> as * だからエラーになるんだと思う
> as * な行と as bs with ... な行を両方用意して、bs ネームスペースで mixin を使えばうまいこといかないかな…

これは、以下で認識あっていますか??
```
@use '../bootstrap/init' as *;
@use '../bootstrap/init' as bs with ($input-focus-color: $color-global);
```

そうだとすると、以下のエラーが出てしまいます。
```
SassError: This module was already loaded, so it can't be configured using "with".
  ╷
2 │ @use '../bootstrap/init' as *;
  │ ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ original load
3 │ @use '../bootstrap/init' as bs with ($input-focus-color: $color-global);
  │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ new load
```

## 2
以下のコードだけにした場合
```
@use '../bootstrap/init' as bs with ($input-focus-color: $color-global);
```